packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -255,11 +293,13 @@ const AccountLimits = ({
                                                 <React.Fragment>
                                                     <tr>
                                                         <AccountLimitsTableCell>
-                                                            {is_appstore ? (
-                                                                <Localize i18n_default_text='Total withdrawal limit' />
-                                                            ) : (
-                                                                <Localize i18n_default_text='Total withdrawal allowed' />
-                                                            )}
+                                                            <Localize
```

Comment:
@yashim-deriv would this cause a problem during translation ?

Reply:
Good catch @henry-deriv will update this :pray: